SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -152,56 +152,35 @@ private static int CreateDeployment(string[] args)
 
                 var createMainDeploymentOp = CreateMainDeploymentAsync(deploymentServiceClient, launchSimPlayerDeployment, projectName, assemblyName, runtimeVersion, mainDeploymentName, mainDeploymentJsonPath, mainDeploymentSnapshotPath, mainDeploymentRegion, mainDeploymentCluster, mainDeploymentTags);
 
-                if (!launchSimPlayerDeployment)
-                {
-                    // Don't launch a simulated player deployment. Wait for main deployment to be created and then return.
-                    Console.WriteLine("Waiting for deployment to be ready...");
-                    var result = createMainDeploymentOp.PollUntilCompleted().GetResultOrNull();
-                    if (result == null)
-                    {
-                        Console.WriteLine("Failed to create the main deployment");
-                        return 1;
-                    }
-
-                    Console.WriteLine("Successfully created the main deployment");
-                    return 0;
-                }
-
-                if (DeploymentExists(deploymentServiceClient, projectName, simDeploymentName))
+                if (launchSimPlayerDeployment && DeploymentExists(deploymentServiceClient, projectName, simDeploymentName))
                 {
                     StopDeploymentByName(deploymentServiceClient, projectName, simDeploymentName);
                 }
 
-                var createSimDeploymentOp = CreateSimPlayerDeploymentAsync(deploymentServiceClient, projectName, assemblyName, runtimeVersion, mainDeploymentName, simDeploymentName, simDeploymentJson, simDeploymentRegion, simDeploymentCluster, simNumPlayers);
-
-                // Wait for both deployments to be created.
-                Console.WriteLine("Waiting for deployments to be ready...");
-                var mainDeploymentResult = createMainDeploymentOp.PollUntilCompleted().GetResultOrNull();
-                if (mainDeploymentResult == null)
+                // HACK: Since we don't have access to dynamic worker flags with the new runtime yet,
```

Comment:
Any indication of how much time this adds to the startup process?
starting deployments in parallel v serial ?

Reply:
I'm afraid, I don't have an answer for that right now. Due to serious disk space limitations I had to push this branch half-way tested (without sim player deployments) as I don't have any more space to check out the example project again. I discussed this with @joshuahuburn beforehand.

I suspect it will roughly double the time for this deployment step, though. If starting a deployment for an already uploaded assembly of a test gym is any indication, then it took 2 minutes, so I suspect this will add around another 2 minutes to the process.

Reply:
it should only add a few minutes. I think it's hard to tell how much time it will actually take, because it varies quite a bit for each launch